backend/test/services/event/event_test_data.py

```diff
@@ -15,36 +15,58 @@
 # Sample Data Objects
 # These sample entities will be used to generate the test data.
 
-event_one = EventEntity(
+event_one = Event(
+    id=1,
     name="Carolina Data Challenge",
     time=date_maker(days_in_future=1, hour=10, minutes=0),
     location="Sitterson Hall Lower Lobby",
     description="Mark your calendars for the 2023 Carolina Data Challenge (CDC)! CDC is UNC's weekend-long datathon that brings together hundreds of participants from across campus, numerous corporate sponsors, tons of free food as well as merch, and hundreds of dollars of prizes!",
     public=True,
-    organization_id=cads.id
+    organization_id=1
```

Comment:
I did this because I ran into an error doing cads.id due to the fact that id can be int | None. I can add a conditional to see if it exists if you think it's better to still do it by calling the object.

Reply:
@atoneyd  No worries - you may be able to try `cads.id!` and see that helps, otherwise we can leave it as hardcoded numbers.